instrumentation/opentelemetry_phoenix/lib/opentelemetry_phoenix.ex

```diff
@@ -183,6 +216,103 @@ defmodule OpentelemetryPhoenix do
     end
   end
 
+  def handle_liveview_event(
+        [:phoenix, _live, :mount, :start],
+        _measurements,
+        meta,
+        _handler_configuration
+      ) do
+    %{socket: socket} = meta
+
+    %{view: live_view} = socket
+
+    attributes = []
+
+    OpentelemetryTelemetry.start_telemetry_span(
+      @tracer_id,
+      inspect(live_view),
+      meta,
+      %{kind: :server}
+    )
+
+    # TODO monitor the process so that if it crashes, we don't lose our span
+    # OpentelemetryMonitor.monitor(OpenTelemetry.Tracer.current_span_ctx())
+
+    OpentelemetryTelemetry.start_telemetry_span(
+      @tracer_id,
+      "#{inspect(live_view)}.mount",
+      meta,
+      %{kind: :server}
+    )
+    |> Span.set_attributes(attributes)
+  end
+
+  def handle_liveview_event(
+        [:phoenix, _live, :handle_params, :start],
+        _measurements,
+        meta,
+        _handler_configuration
+      ) do
+    %{socket: socket} = meta
+
+    %{view: live_view} = socket
+
+    attributes = []
```

Comment:
I think it would be nice to add the `pid` of the Live View process as the `thread.name` following the [attribute semantic conventions](https://opentelemetry.netlify.app/docs/reference/specification/trace/semantic_conventions/span-general/#general-thread-attributes). You can grab the `pid` from the `socket` 🙂

Reply:
Agreed.

Reply:
Also agreed.